packages/connectors/connector-metamask/src/constant.ts

```diff
@@ -0,0 +1,18 @@
+import { ConnectorPlatform, type ConnectorMetadata } from '@logto/connector-kit';
+
+export const defaultMetadata: ConnectorMetadata = {
+  id: 'metamask-universal',
+  target: 'metamask',
+  platform: ConnectorPlatform.Universal,
```

Comment:
📝 This is actually web only. Native will be similar to wechat, 2 connectors with same target.

Reply:
📝 It's also _more_ than MetaMask, its any software implementing epi-1193 provider standards https://eips.ethereum.org/EIPS/eip-1193

Metamask is the most popular but maybe a better way to express this without confusing the user